include/boost/geometry/strategies/geographic/intersection.hpp

```diff
@@ -118,7 +118,6 @@ struct geographic_segments
         return strategy_type(m_spheroid);
     }
 
-    template <typename Geometry>
```

Comment:
What's the reason of changing the interface of geographic intersection strategy? This way it has interface inconsistent with other intersection strategies. Plus it cannot be passed into algorithms (`equals()`) getting distance strategy from intersection strategy without changing the algorithms as well.